README.md

```diff
@@ -14,43 +14,66 @@ Laravel postgis extension
  
  * Geometry functions on the geometry classes (contains(), equals(), distance(), etc… (HELP!))
 
+## Installation
+
+    composer require phaza/laravel-postgis 
+
+Next add the DatabaseServiceProvider to your `config/app.php` file.
+
+    'Phaza\LaravelPostgis\DatabaseServiceProvider',
+
+That's all.
+
 ## Usage
 
-First of all, enable postgis (See note further down)
+A method called enablePostgis() (and disablePostgis()) is included in the Blueprint object.  
+They work on newer postgres installations, but I recommend enabling postgis manually for now.
 
 ### Migrations
 
+Now create a model with a migration by running
+
+    php artisan make:model Location
+
+If you don't want a model and just a migration run
+
+    php artisan make:migration create_locations_table
+
+Open the created migrations with your editor.
+
 ```PHP
+use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
-use Phaza\LaravelPostgis\Schema\Blueprint;
```

Comment:
It's actually better to typehint with "our" Blueprint, as that will give proper code completion in IDE's like PHPStorm and Sublime text for our custom column types.